tests/FakeItEasy.Analyzer.CSharp.Tests/ArgumentConstraintNullabilityMismatchAnalyzerTests.cs

```diff
@@ -0,0 +1,127 @@
+﻿namespace FakeItEasy.Analyzer.CSharp.Tests
+{
+    using System.Collections.Generic;
+    using FakeItEasy.Analyzer.Tests.Helpers;
+    using FakeItEasy.Tests;
+    using Microsoft.CodeAnalysis;
+    using Microsoft.CodeAnalysis.Diagnostics;
+    using Xunit;
+
+    public class ArgumentConstraintNullabilityMismatchAnalyzerTests : DiagnosticVerifier
+    {
+        private const string CodeTemplate = @"using FakeItEasy;
+namespace TheNamespace
+{{

```

Comment:
I saw this and thought my extension was acting up again. Then I saw that it was a template.